--- rubberband-1.8.1/rubberband.pc.in.orig	2012-10-28 11:30:09.000000000 +0100
+++ rubberband-1.8.1/rubberband.pc.in	2016-02-21 21:56:55.376483312 +0100
@@ -1,6 +1,6 @@
 prefix=%PREFIX%
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=%LIBDIR%
 includedir=${prefix}/include
 
 Name: rubberband
--- rubberband-1.9.0/Makefile.in.orig	2021-01-11 21:36:48.112383422 +0100
+++ rubberband-1.9.0/Makefile.in	2021-01-11 21:37:39.002107728 +0100
@@ -195,7 +195,7 @@
 	ln -s $(LIBNAME)$(DYNAMIC_EXTENSION).$(DYNAMIC_FULL_VERSION) $(DESTDIR)$(INSTALL_LIBDIR)/$(LIBNAME)$(DYNAMIC_EXTENSION).$(DYNAMIC_ABI_VERSION)
 	ln -s $(LIBNAME)$(DYNAMIC_EXTENSION).$(DYNAMIC_FULL_VERSION) $(DESTDIR)$(INSTALL_LIBDIR)/$(LIBNAME)$(DYNAMIC_EXTENSION)
 	test -f $(JNI_TARGET) && cp -f $(JNI_TARGET) $(DESTDIR)$(INSTALL_LIBDIR)/$(JNINAME)$(DYNAMIC_EXTENSION) || true
-	sed "s,%PREFIX%,$(PREFIX)," rubberband.pc.in \
+	sed "s,%PREFIX%,$(PREFIX),;s,%LIBDIR%,$(INSTALL_LIBDIR)," rubberband.pc.in \
 	  > $(DESTDIR)$(INSTALL_PKGDIR)/rubberband.pc
 ifeq ($(WITH_PROGRAMS), yes)
 	$(MKDIR) -p $(DESTDIR)$(INSTALL_BINDIR)
