--- rubberband-3.0.0/src/jni/RubberBandStretcherJNI.cpp.orig	2022-07-07 16:59:49.000000000 +0200
+++ rubberband-3.0.0/src/jni/RubberBandStretcherJNI.cpp	2022-07-09 16:00:27.218515481 +0200
@@ -23,7 +23,7 @@
 
 #include "rubberband/RubberBandStretcher.h"
 
-#include "system/Allocators.h"
+#include "common/Allocators.h"
 
 #include <jni.h>
 
--- rubberband-3.0.0/src/common/VectorOpsComplex.cpp.orig	2022-07-07 16:59:49.000000000 +0200
+++ rubberband-3.0.0/src/common/VectorOpsComplex.cpp	2022-07-09 16:00:42.731764772 +0200
@@ -23,7 +23,7 @@
 
 #include "VectorOpsComplex.h"
 
-#include "system/sysutils.h"
+#include "common/sysutils.h"
 
 #include <cassert>
 
