--- rubberband-1.8.2/Makefile.in.orig	2019-04-26 16:04:47.502018628 +0200
+++ rubberband-1.8.2/Makefile.in	2019-04-26 16:42:07.616549552 +0200
@@ -34,6 +34,7 @@
 DYNAMIC_FULL_VERSION	:= 2.1.1
 DYNAMIC_ABI_VERSION	:= 2
 DYNAMIC_LDFLAGS		:= -shared -Wl,-Bsymbolic -Wl,-soname=$(LIBNAME)$(DYNAMIC_EXTENSION).$(DYNAMIC_ABI_VERSION)
+JNI_LDFLAGS		:= -shared -Wl,-Bsymbolic -Wl,-soname=$(JNINAME)$(DYNAMIC_EXTENSION)
 VAMP_LDFLAGS		:= -shared -Wl,-Bsymbolic -Wl,--version-script=vamp/vamp-plugin.map
 LADSPA_LDFLAGS		:= -shared -Wl,-Bsymbolic -Wl,--version-script=ladspa/ladspa-plugin.map
 
@@ -149,7 +150,7 @@
 	$(CXX) -c $(JNI_CXXFLAGS) $(CXXFLAGS) $^ -o $@
 
 $(JNI_TARGET):	$(LIBRARY_OBJECTS) $(JNI_OBJECT)
-	$(CXX) $(DYNAMIC_LDFLAGS) $^ -o $@ $(LIBRARY_LIBS) $(LDFLAGS)
+	$(CXX) $(JNI_LDFLAGS) $^ -o $@ $(LIBRARY_LIBS) $(LDFLAGS)
 
 $(JAR_TARGET):	$(JAVA_SOURCE)
 	$(JAVAC) $^
