--- rubberband-1.8.1/rubberband.pc.in.orig	2012-10-28 11:30:09.000000000 +0100
+++ rubberband-1.8.1/rubberband.pc.in	2016-02-21 21:56:55.376483312 +0100
@@ -1,6 +1,6 @@
 prefix=%PREFIX%
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=%LIBDIR%
 includedir=${prefix}/include
 
 Name: rubberband
--- rubberband-1.8.2/Makefile.in.orig	2019-04-26 05:48:08.645791967 +0200
+++ rubberband-1.8.2/Makefile.in	2019-04-26 05:50:46.468270303 +0200
@@ -189,7 +189,7 @@
 	cp $(LADSPA_TARGET) $(DESTDIR)$(INSTALL_LADSPADIR)
 	cp ladspa/ladspa-rubberband.cat $(DESTDIR)$(INSTALL_LADSPADIR)
 	cp ladspa/ladspa-rubberband.rdf $(DESTDIR)$(INSTALL_LRDFDIR)
-	sed "s,%PREFIX%,$(PREFIX)," rubberband.pc.in \
+	sed "s,%PREFIX%,$(PREFIX),;s,%LIBDIR%,$(INSTALL_LIBDIR)," rubberband.pc.in \
 	  > $(DESTDIR)$(INSTALL_PKGDIR)/rubberband.pc
 
 clean:
